tests/setup-env.js

```diff
@@ -1 +1,3 @@
 import '@testing-library/jest-dom/extend-expect'
+// eslint-disable-next-line import/no-extraneous-dependencies -- need the version from React not an explicitly declared one
+jest.mock('scheduler', () => require('scheduler/unstable_mock'))
```

Comment:
I thought mocking the scheduler wasn't required: https://github.com/reactwg/react-18/discussions/23#discussioncomment-798952

Reply:
It was my understanding that these changes are planned but didn't land yet. At least it was required a week ago and I didn't see anything change. Let me check again and then add a TODO linking the discussion comment.

Reply:
Still warns without the mocked scheduler (message is even outdated):

> Warning: In Concurrent or Sync modes, the "scheduler" module needs to be mocked to guarantee consistent behaviour across tests and browsers. For example, with jest: 
      jest.mock('scheduler', () => require('scheduler/unstable_mock'));
      

Added a TODO comment to make this clear.